earth2studio/utils/interp.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+import torch
+from numpy.typing import ArrayLike
+from scipy.interpolate import LinearNDInterpolator
+from torch import Tensor, nn
+
+
+def latlon_interpolation_regular(
```

Comment:
Confirming this is just a 1:1 copy from corrdiff yeah?

Reply:
Yes, it's just cut/pasted over from corrdiff.